app/views/posts/index.html.erb

```diff
@@ -1,31 +1,17 @@
-<!DOCTYPE HTML>
-<html>
-  <head>
-    <title>User's Posts</title>
-  </head>
-  <body>
-    <header>
-      <%= image_tag url_for(@user.photo) %>
-      <h1><%= @user.name %></h1>
-    </header>
-    <ul>
-      <% @user.posts.each do |post| %>
-        <li>
-          <h3><%= link_to post.title, user_post_path(@user, post) %><h3>
-          <p><%= post.text %></p>
-          <p>Comments: <%= post.comments_counter %> Likes: <%= post.comments_counter %></p>
-          <div>
-            <ul>
-              <% post.comments.each do |comment| %>
-                <li>
-                  <p>Author: <%= comment.author %></p>
-                  <p><%= comment.text %></p>
-                </li>
-              <% end %>
-            </ul>
-        </li>
+<%= link_to "Go Back", :back %>
+<%= render 'shared/user', user: @user %>
+<ul class="posts-list">
+  <% @user.posts.each do |post| %>
+    <li class="posts-list-li">
+      <%= render 'shared/post', post: post %>
+      <% if post.comments.count > 0 %>
+        <ul class="comments-list">
+          <% post.comments.each do |comment| %>
+            <%= render 'shared/comment', comment: comment %>
+          <% end %>
```

Comment:
- [ ] Good job, views are rendering correctly! 👏 However, in this activity we want you to use the methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md), in this case `post.recent_comments` method (to render _5 most recent comments_), like the example below:

```ruby
#app/views/posts/index.html.erb` page

 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```

**Comments of post(s):**
![image](https://user-images.githubusercontent.com/12745474/230155262-8a51f3ea-8da4-451e-bcb0-9d860b590c60.png)

Let's ensure our code fulfils this requirement! 👨‍💻

Reply:
Thanks, I must have missed it!

Reply:
Fixed.